content/en/docs/ops/common-problems/upgrade-issues/index.md

```diff
@@ -102,14 +104,52 @@ spec:
       istio: ingress-gateway
 {{< /text >}}
 
-Use the equivalent ingress gateway proxy configuration annotation:
+Use the equivalent ingress gateway pod proxy configuration annotation:
 
 {{< text yaml >}}
 metadata:
   annotations:
     "proxy.istio.io/config": '{"gatewayTopology" : { "numTrustedProxies": 1 }}'
 {{< /text >}}
 
+### Use gateway topology to enable PROXY protocol on the ingress gateways
```

Comment:
```suggestion
### Use gateway topology to enable proxy protocol on the ingress gateways
```
Checked `proxy protocol` in the entire repo. Is it necessary to capitalize all letters with `PROXY`?

![image](https://github.com/istio/istio.io/assets/79828097/bbbec3b6-6960-4a4f-9702-85dfcca5d792)

Reply:
I agree when referencing the PROXY Protocol as defined, we should capitalize to distinguish it similar to the RFC.